src/main/java/com/hyunec/cosmicbaseballinit/domain/baseball/model/PlateAppearances.java

```diff
@@ -1,19 +1,59 @@
 package com.hyunec.cosmicbaseballinit.domain.baseball.model;
 
+import lombok.Getter;
 import org.springframework.stereotype.Component;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import static com.hyunec.cosmicbaseballinit.domain.baseball.model.Batting.BALL;
 import static com.hyunec.cosmicbaseballinit.domain.baseball.model.Batting.STRIKE;
+import com.hyunec.cosmicbaseballinit.domain.baseball.exception.ExceptionMessage;
+
+@Getter
 
 @Component
 public class PlateAppearances {
 
     private final List<Batting> battings = new ArrayList<>();
+    private final Map<String, String> player = new HashMap<String, String>();
+
+    private static PlateAppearances pa;
+
+    Integer totalOut = 0;
+    boolean isPlaying = false;
+
+    public PlateAppearances() {
+        player.put("1player", "N"); //N 게임 미진행, U: 게임진행중, E: 게임끝
+    }
+
+    public static synchronized PlateAppearances getInstance() {
+        if(pa == null) {
+            pa = new PlateAppearances();
+        }
+        return pa;
+    }
+
+//    public boolean getPlaying() {
+//        return this.isPlaying;
+//    }
+
+    public void setPlaying(boolean is) {
+        this.isPlaying = is;
+    }
+
+//    public Integer gettotalOut() {
+//        return this.totalOut;
+//    }
+
+    public void settotalOutCount() {
+        this.totalOut++;
+    }
```

Comment:
```suggestion
    public void setTotalOutCount() {
        this.totalOut++;
    }
```

이런 오타는 이력서의 맞춤법이 틀림과 같습니다.
필히 신경 써주세요.

Reply:
넵 감사합니다. 지적감사드립니다^^